app/scripts/translations.js

```diff
@@ -14,7 +14,7 @@ angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
     gettextCatalog.setStrings('nl', {});
```

Comment:
@petrjasek you are right of course. I corrected what I could find and I will pass on the info to the translators.